src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -40,7 +39,7 @@ export class MongoCollectionTreeItem implements IAzureParentTreeItem {
 			};
 		});
 
-		const result = await this.collection.bulkWrite(operations);
+		await this.collection.bulkWrite(operations);
```

Comment:
Have you verified this result is something okay to ignore?

Reply:
I placed this line back in [this commit](https://github.com/Microsoft/vscode-cosmosdb/blame/21c85e9cc850a7ed188abca3a0d36cb2fcabfb0b/src/mongo/nodes.ts#L205). I placed it solely for debugging the code. 
Tested it again, see no issues atm.

Reply:
I don't think we should ignore this. You could at least print something like this to the output window: `Successfully updated ${result.modifiedCount} document(s).`